back/engines/free/user_confirmation/app/serializers/user_confirmation/extensions/web_api/v1/user_serializer.rb

```diff
@@ -0,0 +1,16 @@
+module UserConfirmation
+  module Extensions
+    module WebApi
+      module V1
+        module UserSerializer
+          def self.included(base)
+            base.class_eval do
+              base.attributes :email_confirmed_at
```

Comment:
Just to be sure, should everyone be allowed to see that someone confirmed their email, or should this attribute only be visible to admins and the user himself?

Reply:
The attribute is actually not necessary anymore, I was using this before adding `requires_confirmation`. Will remove 👍